packages/botonic-react/src/components/text.jsx

```diff
@@ -1,8 +1,29 @@
 import React from 'react'
 import { Message } from './message'
 
+const serialize = textProps => {
+  let text = textProps.children
+  /* As text message can have multiple children ( a text with buttons or quickreplies )
+    we only want to store the text, we don't want to store the buttons or quickreplies.
```

Comment:
We don't want to store quick replies, ok. But buttons? I think we should store them, no?

Reply:
Yes, we want to store quickreplies and buttons too, but we don't want to decompose here in the text, because in the webchat, we are already dealing with the decomposition of quickreplies and buttons.